Resources/Prototypes/Roles/Jobs/Engineering/chief_engineer.yml

```diff
@@ -12,7 +12,7 @@
       time: 21600 #6 hrs
     - !type:DepartmentTimeRequirement
       department: Engineering
-      time: 36000 #10 hrs
+      time: 75600 #21 hrs
     - !type:OverallPlaytimeRequirement
       time: 144000 #40 hrs
```

Comment:
6 hours of AT + 6 hours of SE = 21 hours of engineering?

Reply:
You need 15 hours to unlock Atmospheric tech.

You need 6 hours of atmospheric tech to unlock CE.

15 + 6 = 21.

You unlock CE before senior engineer.

Senior requires 60 hours of playtime while CE only requires 40

Reply:
I don't think the role timers were designed with that kind of additive math in mind.

While it's true that it currently takes 15 hours in engineering to unlock atmos tech, and 6 hours of atmos tech to unlock CE, that doesn't mean the engineering department time should necessarily be the sum total. The point of the department role time requirement is that if the other role time requirements were changed somehow, there would still be a minimum 10 hours of department play required. It's sort of a guardrail to make sure that no matter what changes are made to the specific role times, that there's still 10 hours. If that's not appropriate, perhaps it would be better to remove the engineering department time requirement rather than duplicate the same number in two places.

Therefore I propose to close this.